.travis.yml

```diff
@@ -52,8 +38,29 @@ after_script:
 - bash <(curl -s https://codecov.io/bash)
 jobs:
   allow_failures:
-  - env: NODE_VERSION=12.12.0
+  - env: NODE_VERSION=12.12.0	
+  - env: POSTGRES_MAJOR_VERSION=12 PARSE_SERVER_TEST_DB=postgres
   include:
+  - stage:
+    addons:
+      postgresql: '11'
+      apt:
+        packages:
+          - postgresql-11-postgis-2.5
+          - postgresql-11-postgis-2.5-scripts
+    env: POSTGRES_MAJOR_VERSION=11 PARSE_SERVER_TEST_DB=postgres
+    before_install: bash scripts/before_install_postgres.sh
+    before_script: bash scripts/before_script_postgres.sh
+  - stage:
+    addons:
+      postgresql: '12'
```

Comment:
Can you remove PG 12 from here? Our build matrix is all ready at the max for concurrent jobs (5) and is taking ~19 minutes to build.

Everything looks good! Lets see how it goes.

Reply:
Done